05-jupyter-ecosystem.md

```diff
@@ -336,8 +336,51 @@ time to complete.
 
 ### Explaining each cell
 
-A good rule of thumb is to always include a markdown cell above every code cell.
-
+A good rule of thumb is to always include a markdown cell above every
+code cell. In fact, instead of adding comments in code, consider
```

Comment:
Perhaps: Consider moving the comments for a code block into a markdown cell either directly above or below the code cell. Comments in a markdown cell often read much better and give you more flexibility in discussing or describing the code.

Reply:
Good!